modules/azure/function_app_linux_managed_identity/main.tf

```diff
@@ -176,11 +160,12 @@ resource "azurerm_monitor_diagnostic_setting" "diagnostic_setting" {
   target_resource_id         = azurerm_linux_function_app.function_app.id
   log_analytics_workspace_id = var.log_analytics_workspace_id
 
-  dynamic "enabled_log" {
-    for_each = data.azurerm_monitor_diagnostic_categories.diagnostic_categories[0].log_category_types
+  dynamic "log" {
```

Comment:
Is there any reason why this and log_category_types are reverted back to the deprecated attributes?